plugins/CMakeLists.txt

```diff
@@ -73,4 +77,5 @@ endif()
 
 if(WASMEDGE_PLUGIN_ZLIB)
   add_subdirectory(wasmedge_zlib)
+  
```

Comment:
Why do you add a blank line here?

Reply:
I believe it is how I have set up my code editor to add a blank line at the end of a file, I will remove the setting if that is not accepted for this repository.